fluid/PaddleCV/image_classification/reader.py

```diff
@@ -96,7 +96,7 @@ def random_color(img, lower=0.5, upper=1.5):
     return img
 
 
-def process_image(sample, mode, color_jitter, rotate):
+def process_image(sample, mode, color_jitter, rotate, normalize):
```

Comment:
why not set normalize=True as default option?

Reply:
norm image in graph can get better performance.